modules/ossm-config-openshift-monitoring-ambient-mode.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/metrics/ossm-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-ambient-mode_{context}"]
+= Configuring OpenShift Monitoring with Service Mesh ambient mode
+
+[role="_abstract"]
+You can integrate {SMProductName} with user-workload monitoring to enable observability in your service mesh ambient mode. User-workload monitoring provides access to essential built-in tools and is required to run Kiali, the dedicated console for {istio}. 
+
+.Prerequisites
+
+* You have installed the {SMProductName} Operator.
+* You have enabled the user-workload monitoring.
++
+[NOTE]
+====
+You can enable user-workload monitoring by applying the `ConfigMap` change for metrics integration. For more information, see link:https://docs.openshift.com/container-platform/latest/observability/monitoring/enabling-monitoring-for-user-defined-projects.html[Enabling monitoring for user-defined projects].
+====
+* You have deployed the Bookinfo application in ambient mode to use the following example. For more information, see https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.2/html/installing/ossm-istio-ambient-mode#ossm-deploying-bookinfo-application-istio-ambient-mode_ossm-istio-ambient-mode[Deploying the Bookinfo application in Istio ambient mode].
```

Comment:
I don't think the step of deploying a Bookinfo application is a Prerequisite.
We can move this Bookinfo step from Prerequisties section to a Validation section.